code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -544,97 +544,85 @@
 /datum/spellbook_entry/summon
 	name = "Summon Stuff"
 	category = "Rituals"
+	limit = 1
 	refundable = FALSE
 	buy_word = "Cast"
 
-/datum/spellbook_entry/summon/CanBuy(mob/living/carbon/human/user,obj/item/spellbook/book)
-	return ..() && !times
+/datum/spellbook_entry/summon/Buy(mob/living/carbon/human/user, obj/item/spellbook/book)
+	log_spellbook("[key_name(user)] cast [src] for [cost] points")
+	SSblackbox.record_feedback("tally", "wizard_spell_learned", 1, name)
+	times++
+	return TRUE
 
 /datum/spellbook_entry/summon/ghosts
 	name = "Summon Ghosts"
 	desc = "Spook the crew out by making them see dead people. Be warned, ghosts are capricious and occasionally vindicative, and some will use their incredibly minor abilities to frustrate you."
 	cost = 0
 
 /datum/spellbook_entry/summon/ghosts/Buy(mob/living/carbon/human/user, obj/item/spellbook/book)
-	log_spellbook("[key_name(user)] cast [src] for [cost] points")
-	SSblackbox.record_feedback("tally", "wizard_spell_learned", 1, name)
-	new /datum/round_event/wizard/ghost()
-	times++
-	to_chat(user, span_notice("You have cast summon ghosts!"))
+	summon_ghosts(user)
 	playsound(get_turf(user), 'sound/effects/ghost2.ogg', 50, TRUE)
-	return TRUE
+	return ..()
 
 /datum/spellbook_entry/summon/guns
 	name = "Summon Guns"
 	desc = "Nothing could possibly go wrong with arming a crew of lunatics just itching for an excuse to kill you. There is a good chance that they will shoot each other first."
 
 /datum/spellbook_entry/summon/guns/IsAvailable()
-	if(!SSticker.mode) // In case spellbook is placed on map
-		return FALSE
-	if(istype(SSticker.mode, /datum/game_mode/dynamic)) // Disable events on dynamic
+	// Summon Guns requires 100 threat.
+	if(istype(SSticker.mode, /datum/game_mode/dynamic))
```

Comment:
Remove this check entirely, Dynamic will forever be the last gamemode, I was very close to killing it but got proeccupied.